### ISSUE TITLE
Need R Shiny integration with RVision package

### ISSUE THREAD

**Issue by mdhawk** [swarm-lab/Rvision#32]
I don't find a single reference online if RVision can be integrated with R Shiny. If it is currently not supported then can we at least get this functionality in the near future.

Thanks

**Comment by sjmgarnier** [comment]
@mdhawk It depends what you mean by "integrated". For instance, [trackR](https://github.com/swarm-lab/trackR) uses Shiny and Rvision for doing video tracking of animals. But if you're hoping to have live view integrated inside a Shiny app, that won't be possible before some time due to the absence of easy interface between OpenCV and a web browser.

**Comment by mdhawk** [comment]
@sjmgarnier Yes I was hoping there is some function or online reference that could directly integrate live view inside a Shiny app. If it were possible would have been really great. Anyways thanks for the quick and clear response.

**Comment by sjmgarnier** [comment]
@mdhawk It's just too complicated for me at the moment. It requires setting up some form of streaming server in the background in order to push the video to the web browser.
